src/index.js

```diff
@@ -37,14 +41,16 @@ app.use(
     origin: [`http://localhost:3000`, process.env.SLACKBOT_FE_URL]
   })
 );
+
+// Add channels 
 // app.use("/", router);
 // app.use(errorHandler);
+app.use(bodyParser());
 app.use("/channels", channelRouter);
 // // app.get("/", );
-
+app.use("/new-poll", newPollRouter);
 app.use("/previous-polls", previousPollsRouter);
```

Comment:
this is totally a style thing, my personal preference would be to expose the following two routes:

```
GET /polls
POST /polls
```

the GET would do what your `previous-polls` route is currently doing, and the POST would do what your `new-polls` route is doing. 

This brings the app more inline with common RESTful practices...they're referring to the same resource, the differentiator is in the HTTP method (or "verb") used to access them. 

However, this is totally up to you, what you have here will work too!

Reply:
Aha, that actually makes more sense and also make the structure dry. I have made the change. Thank you!